supporting_content/metrics/amortizedcost.md

```diff
@@ -0,0 +1,71 @@
+| ----------------------- | ------------------------ | ----------- | ----- | ----------- | -------------- |
+| Jan 1 2023 - Jan 1 2024 | Jan 1 2023 - Jan 31 2023 | Purchase    | 3     | 26.69589041 | 0              |
+
+
+###### Note 
+*Billed cost is showing the total cash outlay for the reservation. Also note there is no initial RI fee representing the cost of the RI for the applicable period.*
+
+
+
+##### Also on each hour a line item is added as follows
+
+| Charge period           | Billing period           | Charge Type | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | ----------- | ----- | ----------- | -------------- |
+| Jan 1 2023 - Jan 2 2023 | Jan 1 2023 - Jan 31 2023 | Usage       | 3     | 0           | 3.81369863     |
+
+
+
+###### Note: 
+*The amortization is equal to the (Upfront fee / (number of normalized units reserved x periods) x normalized line item units consumed).*
+
+___
+
+##### If the RI is unused during a period, for example another ineligible instance type is used:
+
+
+| Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usag         | 3     | 5.544       | 5.544          |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 3     | 0           | 3.81369863     |
+
+
+
+###### Note: 
+*For the On Demand usage the Billed cost and Amortized cost are the same with an Amortization of Zero. Since the RI went completely unused during the period there is an unused RI usage line item with the an amortization equal to (Upfront fee / (number of normalized units reserved x periods) x normalized line item units. Amortized Cost is equivalent to Amortized since this is a full upfront reservation*
+
+---
+
+##### If the RI is partially unused during a period, for example a scale down occurs:
+
+
+| Charge period           | Billing period           | Charge Type     | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | --------------- | ----- | ----------- | -------------- |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | OD Usag         | 3     | 5.544       | 5.544          |
+| Jan 4 2023 - Jan 5 2023 | Jan 1 2023 - Jan 31 2023 | Unused RI Usage | 3     | 0           | 3.81369863     |
+
+
+
+###### Note: 
+*For the RI Usage the Billed cost is zero as this RI is full upfront. It is especially important to note that the usage units must be normalized for this formula to be applicable in cases of size flexibility (e.g. c7g.large normalized units is 4, c7g.medium is 2). Amortized cost is the same as amortization because the upfront fee covers all of the compute usage expense for this line item.*
```

Comment:
Do we need to define RI as "Reserved Instance" somewhere?

Reply:
Yeah let's use the full name.

Reply:
Updated now.